### ISSUE TITLE
[Task]: Context Switching Phase 3

### ISSUE THREAD

**Issue by hetunandu** [appsmithorg/appsmith#18014]
## Debugger States

- [ ] Filters applied on the logs and errors (text/entity/severity) should be persisted at the page level
- [ ] Scroll Position of the debugger
- [ ] Collapsibles items should remain in the same state while navigating

## Tabs Widget (blocked by #1282)
- [x] Context Switching for Tabs
- [x] CMD click for Tabs

**Comment by rohan-arthur** [comment]
won't fix - project cancelled
